wordpress-mobile/WordPress-iOS
# Add "You hit a milestone 🚀" title to view milestone push notifications

Fixes #NA

**To test:**

Pre-requisites:
- Make sure the api calls from your device point to your sandbox, and have the latest version of milestone notifications in there
- Enable the `milestoneNotifications` feature flag
- Open the push notifications console on mission control dev, from a browser that points to the same sandbox

Test execution:
- build/run this branch and background the app
- on the push notification console, select an existing milestone notification (or trigger a new one from wp shell)
- From the applications, select wp iOS dev; optionally, select a device where you want to send the push (if you don't, it will send it to all your devices where the selected app is installed)
- Hit "Send Notification"
- Make sure the notification content looks like the example below
- Tap on it, and make sure it takes to the related notification
- Select other types of notifications from the list, send them and make sure they look/behave as expected

<p align=center>
<img width="450" src="https://user-images.githubusercontent.com/34376330/109353020-67866380-7841-11eb-87f0-ca15b1c49585.png">
</p>

PR submission checklist:
